### DISCUSSION TITLE
I guess orm_mode is not working with generic models

### DISCUSSION THREAD

**Discussion by danielmoessner** [pydantic/pydantic#5816] [Pydantic V2]
```
class OutputPagination(GenericModel, Generic[P]):
    items: list[P]
    total: int

class OutputMedic(BaseModel):
    id: UUID
    name: str
    title: str | None
    gender: str

    class Config:
        orm_mode = True

model = OutputPagination[OutputMedic]
```

This does not work for me with Medic not being a dict. Later I might have time to update this example and turn it into an issue or if somebody wants to take a look at it feel free.

**Comment by dmontagu** [comment]
Have you tried setting `orm_mode=True` on OutputPagination?

**Reply by danielmoessner** [reply]
I haven't but I put in a dict to OutputPagination. I only put instances for items.

**Reply by dmontagu** [reply]
If you can share a full code sample demonstrating the issue we can look into it. I'm thinking it may be caused by the fact that it's a list. In general, I would expect this is fixed in v2, but this is worthy of a bugfix on v1 if we can reproduce and resolve the issue without major changes.
